XCUITests/FxScreenGraph.swift

```diff
@@ -483,6 +486,15 @@ func createScreenGraph(for test: XCTestCase, with app: XCUIApplication) -> MMScr
         screenState.press(app.tables["History List"].cells.element(boundBy: 2), to: HistoryPanelContextMenu)
         screenState.tap(app.cells["HistoryPanel.recentlyClosedCell"], to: HistoryRecentlyClosed)
         screenState.tap(app.buttons["Done"], to: HomePanelsScreen)
+        screenState.gesture(forAction: Action.ClearLastHourHistory) { userState in
+            app.tables["History List"].staticTexts["Clear Recent History…"].tap()
+            app.sheets["Clearing Recent History will remove history, cookies, and other browser data."].buttons["The Last Hour"].tap()
```

Comment:
This is a long label to be included in the ScreenGraph :( I have checked and can't not find easily how to add an id. We could add a bug for asking that and in the meantime use something like:
`app.sheets.element(boundBy: 0).buttons["Today"]`
With a comment to know where it is tapping...wdyt?

Reply:
yup, SGTM.